### PR TITLE
remove customresourcedefinitions from boskos ClusterRole

### PR DESCRIPTION
**Pull request by ixdy**
Fixes https://github.com/kubernetes/test-infra/issues/16281.

cc @alvaroaleman @stevekuznetsov

### CODE REVIEW

**Comment by ixdy** [comment]
cc @BenTheElder

File: prow/cluster/boskos.yaml [hunk 1]
```diff
@@ -98,7 +95,7 @@ metadata:
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: crd-creator
+  name: boskos
```
Comment (alvaroaleman) [inline prow/cluster/boskos.yaml]: Someone needs to manually delete the old `crd-creator` role after this gets deployed

Comment (ixdy) [reply prow/cluster/boskos.yaml]: @BenTheElder is the current on-call

Comment (BenTheElder) [reply prow/cluster/boskos.yaml]: presumably after the next bump? I've had a full schedule today and haven't bumped yet.

Comment (ixdy) [reply prow/cluster/boskos.yaml]: This has been deployed now, so the old role binding (named `crd-creator`) can be deleted.

Comment (BenTheElder) [reply prow/cluster/boskos.yaml]: this is done

**Pull request merged**